addons/event/views/event_registration_views.xml

```diff
@@ -327,6 +353,29 @@
         </field>
     </record>
 
+    <record id="event_registration_action_stats_from_event" model="ir.actions.act_window">
+        <field name="name">Registration statistics</field>
+        <field name="res_model">event.registration</field>
+        <field name="view_mode">graph,pivot,kanban,tree,form</field>
+        <field name="domain">[('event_id', '=', active_id)]</field>
+        <field name="context">
+            {
```

Comment:
Can we use the same syntax as the other one? Meaning the opening bracket is on the same line as the field tag?